.vscode/launch.json

```diff
@@ -0,0 +1,30 @@
+{
+  // Use IntelliSense to learn about possible attributes.
+  // Hover to view descriptions of existing attributes.
+  // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+  "version": "0.2.0",
+  "configurations": [
+    {
+      "name": ".NET Core Launch (console)",
+      "type": "coreclr",
+      "request": "launch",
+      "program": "${workspaceFolder}/artifacts/bin/redist/Debug/dotnet/sdk/10.0.100-dev/dotnet.dll",
+      "args": [
+      ],
+      "env": {
+        "MSBuildExtensionsPath": "${workspaceFolder}/artifacts/bin/redist/Debug/dotnet/sdk/10.0.100-dev",
+        "MSBuildSDKsPath": "${workspaceFolder}/artifacts/bin/redist/Debug/dotnet/sdk/10.0.100-dev/Sdks",
```

Comment:
So, these paths will break at each major version update?

Reply:
quite possibly, but there's not a _great_ way to make this data-driven that I can see. normally you'd have devkit or something generate this for you, but since our setup is so particular their mechanisms don't work for us

Reply:
Yes, this will have to be updated with each major version.

Reply:
We can create a `tasks.json` with a `preLaunchTask` that uses the .NET Install Tool API `dotnet.recommendedVersion` to get the latest version of .NET and prepopulate the hard-coded value

Reply:
this is a good idea, but doesn't easily work because that command doesn't return just a single string. I think we take the hit for now and just do tiny bumps as we switch branding.